src/main/java/com/example/notes/mapper/NoteMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.example.notes.mapper;
+
+import com.example.notes.dto.NoteDto;
+import com.example.notes.entity.NoteEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class NoteMapper {
+
+    public List<NoteDto> mapEntityToDto(List<NoteEntity> noteEntityList) {
+        return noteEntityList.stream()
+                .map(noteEntity -> new NoteDto(
+                        noteEntity.getContent(),
+                        noteEntity.getCreatedAt()))
```

Comment:
We must not create the DTO instances with the new keyword. At some point in time, this DTO will grow in size and will have more fields of the same type. We might accidentally swap properties in the constructor and incorrectly create an instance (creating instances with a constructor is more error-prone in general once the amount of properties is large).

Reply:
Ok, replaced with builder